src/NodaTime/LocalDate.cs

```diff
@@ -639,7 +626,7 @@ public LocalDate WithCalendar([NotNull] CalendarSystem calendar)
         public LocalDate PlusWeeks(int weeks) => DatePeriodFields.WeeksField.Add(this, weeks);
 
         /// <summary>
-        /// Returns the next <see cref="LocalDate" /> falling on the specified <see cref="IsoDayOfWeek"/>.
+        /// Returns the next <see cref="LocalDate" /> falling on the specified <see cref="DayOfWeek"/>.
```

Comment:
I think this is supposed to be a reference to the type, not the property, but I guess the fact that you've changed it suggests that this should actually be `<see cref="NodaTime.IsoDayOfWeek"/>`.